TFT/src/User/API/UI/list_item.c

```diff
@@ -156,7 +156,6 @@ void ListItem_Display(const GUI_RECT* rect, uint8_t iconchar, ICON_POS pos, uint
   if (iconchar != ICONCHAR_BACKGROUND){
     GUI_SetColor(ICON_COLOR[iconchar]);
     GUI_DispString(icon_x, icon_y,IconCharSelect(iconchar));
-    GUI_DispString(icon_x + BYTE_HEIGHT + 3, icon_y , label);
```

Comment:
@HappyCodingRobot  removing this line will also stop the display of text in list items where text is declared. the bug is fixed through #246 . please check and revert.